### PR TITLE
real time search

### PR DESCRIPTION
**Pull request by 0x5eba**

### CODE REVIEW

**Comment by nicomazz** [comment]
Looks good, but maybe it is better to add a delay before performing the new query. What if someone types very fast, and each query takes 2 seconds?

File: src/App.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ import MyToolbar from "./components/Toolbar";
 import { withStyles } from "@material-ui/core/styles";
 import AppBar from "@material-ui/core/AppBar";
 
-
+var delayTimer;
```
Comment (nicomazz) [inline src/App.js:38]: I'm not sure about js global variables scoping, but maybe it is better to rename this to something like "searchDelayTimer"
```suggestion
var searchDelayTimer;
```

Can you also move it just before `class App` please?

**Review by nicomazz — APPROVED** [review]
Looks good!
Have you tried if it actually works?

**Comment by 0x5eba** [comment]
Yes, except for the last commit

**Pull request merged**